.github/workflows/dependabot-bundler.yaml

```diff
@@ -0,0 +1,29 @@
+name: Dependabot Bundler
+
+on: workflow_dispatch
+
+jobs:
+  bundler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18.x
+      - name: Cache go-build and mod
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build/
+            ~/go/pkg/mod/
+          key: go-${{ hashFiles('go.sum') }}
+          restore-keys: |
+            go-
+      - name: Install Dependabot Bundler
+        run: |
+          go install github.com/Skarlso/dependabot-bundler@v0.0.7
```

Comment:
Do we want to fork this to the `weaveworks` org?

Reply:
We can. I don't know the consensus on that.